app/database/MetricsDB.scala

```diff
@@ -21,8 +23,10 @@ class MetricsDB(val db: Database) {
   def getForks: Seq[Fork] = await(db.run(forksTable.result))
   def insertFork(fork: Fork): Int = await(db.run(forksTable += fork))
 
+  val dayOfWeek = SimpleFunction.binary[String, Timestamp, Timestamp]("date_trunc")
+
   def getStartedInSystem(implicit filters: MetricsFilters): Seq[(DateTime, Int)] =
```

Comment:
Really nice work. Maybe add a comment in here just to clarify we need to group by day?